app/dao/notifications_dao.py

```diff
@@ -354,18 +354,27 @@ def dao_update_sms_notification_delivery_status(
     """
 
     if notification_type == SMS_TYPE:
+        stmt_values = {
+            'status': new_status,
+            'status_reason': new_status_reason,
+            'segments_count': segments_count,
+            'cost_in_millicents': cost_in_millicents,
+        }
+
+        if new_status in FINAL_STATUS_STATES:
+            notification = db.session.get(Notification, notification_id)
+            stmt_values['_personalisation'] = encryption.encrypt(
+                {k: '<redacted>' for k in notification.personalisation}
+            )
+
         stmt = (
             update(Notification)
             .where(Notification.id == notification_id)
             .where(sms_conditions(new_status))
-            .values(
```

Comment:
I know Chris's recent PR also touched this query as well. I don't think he's merged his PR yet but there may be some conflicts later.

Reply:
NVM!